js/app.js

```diff
@@ -8,14 +8,17 @@ function Project(name, url, about, image) {
   Project.all.push(this);
 }
 
-Project.prototype.render = function() {
-  var $template = $('#template').clone();
-  $template.find('h2').text(this.name);
-  $template.attr('id', '').attr('href', this.url);
-  $template.find('img').attr('src', this.image);
-  $template.find('p').text(this.about);
-  $('#projectSection').prepend($template);
-};
+
+Project.prototype.toHtml = function() {
+  var projectTemplate = Handlebars.compile($('#article-template').html());
+  var context = {
+    name: this.name,
+    url: this.url,
+    about: this.about,
+    image: this.image
+  }
```

Comment:
Since `.toHtml` is a method on an instance of a Project object, you do not need to create a new object here. You can just use the instance of the object that this method is acting on as the context.